app/controllers/application_controller.rb

```diff
@@ -32,22 +37,38 @@ def default_url_options(options = {})
 
     if current_organization.present? && !options.key?(:organization_id)
       options[:organization_id] = current_organization.to_param
-    elsif current_user && !current_user.super_admin? && current_user.organization.present?
+    elsif current_role.name == "org_admin"
       options[:organization_id] = current_user.organization.to_param
-    elsif current_user&.super_admin?
+    elsif current_role.name == "super_admin"
       # FIXME: This *might* not be the best way to approach this...
       options[:organization_id] = "admin"
     end
     options
   end
 
+  def dashboard_path_from_role
```

Comment:
```suggestion
  def dashboard_path(role: role)
```
or maybe we can pass in the user?

Reply:
It's a controller method, meaning that both `current_user` and `current_role` should be in scope at all times. Not sure what we would gain by passing the value in?

Reply:
My main suggestion is about clarity, could we change it to `dashboard_path_for_current_role`? Out of context, I think `from_role` doesn't make sense.